src/routes/safe/components/Transactions/TxList/BatchExecuteButton.tsx

```diff
@@ -0,0 +1,95 @@
+import { Theme, Tooltip, withStyles } from '@material-ui/core'
+import { ReactElement, useCallback, useContext, useMemo } from 'react'
+import { useSelector } from 'react-redux'
+import Button from 'src/components/layout/Button'
+import { OnboardingWidget } from 'src/components/OnboardingWidget'
+import { getBatchableTransactions } from 'src/logic/safe/store/selectors/gatewayTransactions'
+import { isTxPending } from 'src/logic/safe/store/selectors/pendingTransactions'
+import { AppReduxState } from 'src/store'
+import { md, sm } from 'src/theme/variables'
+import styled from 'styled-components'
+import { BatchExecuteHoverContext } from './BatchExecuteHoverProvider'
+
+interface BatchExecuteButtonProps {
+  onClick: () => void
+}
+
+const HELP_STORAGE_KEY = 'batchExecutionButtonSeen'
+
+export const BatchExecuteButton = ({ onClick }: BatchExecuteButtonProps): ReactElement => {
+  const batchableTransactions = useSelector(getBatchableTransactions)
+  const hoverContext = useContext(BatchExecuteHoverContext)
+  const store = useSelector((state: AppReduxState) => state)
+  const hasPendingTx = useMemo(
+    () => batchableTransactions.some(({ id }) => isTxPending(store, id)),
+    [batchableTransactions, store],
+  )
+  const isBatchable = batchableTransactions.length > 1
+  const isDisabled = !isBatchable || hasPendingTx
+
+  const handleOnMouseEnter = useCallback(() => {
+    hoverContext.setActiveHover(batchableTransactions.map((tx) => tx.id))
+  }, [batchableTransactions, hoverContext])
+
+  const handleOnMouseLeave = useCallback(() => {
+    hoverContext.setActiveHover()
+  }, [hoverContext])
+
+  return (
+    <OnboardingWidget
+      text="Queued transactions can now be executed in batches!"
+      widgetLocalStorageId={HELP_STORAGE_KEY}
+    >
+      <ButtonWrapper>
+        <StyledTooltip
+          interactive
+          title={
+            isDisabled
+              ? 'Batch execution is only available for transactions that have been fully signed and which are strictly sequential in Safe Nonce.'
```

Comment:
I'm not 100% sure on this one but @johannesmoormann brough this up with me.

```suggestion
              ? 'Batch execution is only available for transactions that have been fully signed and are strictly sequential in Safe Nonce.'
```